Live/Classes/Player.cpp

```diff
@@ -100,3 +100,7 @@ void Player::stopMove() {
     m_sprite->stopAllActions();
     m_sprite->setDisplayFrame(m_anim_map[m_orientation].at(STATIONARY_INDEX));
 }
+
+Rect Player::getHitbox() {
+    return Rect(getPosition().x-2,getPosition().y-24, 4, 12);
```

Comment:
Magic numbers here again, also wouldn't it make more sense to change `Obstacle` to `RigidObject`/`RigidBody` and have Player also inherit from that, so we dont have to reimplement `get/setHitbox`?

Reply:
can you make another ticket for this? lol